src/Npgsql/NpgsqlCommand.cs

```diff
@@ -556,13 +556,15 @@ void DeriveParametersForQuery()
         /// Creates a server-side prepared statement on the PostgreSQL server.
         /// This will make repeated future executions of this command much faster.
         /// </summary>
+        /// <param name="cancellationToken">The token to monitor for cancellation requests. The default value is <see cref="CancellationToken.None"/>.</param>
```

Comment:
There is no default value. Let's cleanup that here and below.

Reply:
Removed documentation note for default for default values, except for newly-introduced async methods which actually have a default (e.g. PrepareAsync).